src/components/UserMasterPanel.js

```diff
@@ -1,8 +1,11 @@
-import React, { useEffect } from "react";
+import React, { useEffect, useState } from "react";
 import { connect, useDispatch } from "react-redux";
 
 import { withTheme, withStyles } from "@material-ui/core/styles";
-import { Grid, Divider, Typography } from "@material-ui/core";
+import { Grid, Divider, Typography, Button, InputAdornment, IconButton, Box } from "@material-ui/core";
+import VisibilityIcon from '@material-ui/icons/Visibility';
+import VisibilityOffIcon from '@material-ui/icons/VisibilityOff';
+import { passwordGenerator } from "../helpers/passwordGenerator";
```

Comment:
Imports should be categorized into three distinct sections (external libraries, ui imports, internal imports) and each section should be separated by new lines. Could you move this import to the 3rd section?